.github/workflows/ci.yml

```diff
@@ -17,16 +17,18 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         # TODO: update each time a new major Java version is released.
-        java: [8, 11, 14]
+        java-version: [8, 11, 14]
```

Comment:
On 7.x, I do not think we need 14. At this point, it is a waste of resources.

Reply:
I can delete it in spite of it is not the matter for this change, wdyt?

Reply:
@triceo can you please check? As I said I would treat jdk14 discussion in a different thread (PR), but up to you...

Reply:
```suggestion
        java-version: [8, 11]
```